app/src/main/java/com/breadwallet/entities/IntroLanguageResource.kt

```diff
@@ -7,8 +7,8 @@ class IntroLanguageResource {
     private val EUR: String = "EUR"
     private val RMB: String = "RMB"
     private val JPY: String = "JPY"
-    fun loadResources() : Array<IntroLanguage>{
-        return arrayOf<IntroLanguage> (
+    fun loadResources(): Array<IntroLanguage> {
```

Comment:
Was this your linter or just your notice/fix?...I love linter/autoformatters

Reply:
I see, I just using reformat code from Android Studio